crates/router/src/routes/app.rs

```diff
@@ -661,11 +664,15 @@ impl Cache {
 }
 
 pub struct PaymentLink;
-
+#[cfg(feature = "olap")]
```

Comment:
the whole of payment link is under olap? initiate payment link should be under oltp right?

Reply:
payment link should ideally be created from dashboard only, Only initiate payment link should be called from browser Once the user launches the url